### ISSUE TITLE
Getting an error calling model.score

### ISSUE THREAD

**Issue by Industrial** [javascriptdata/scikit.js#234]
Hi!

Maybe I did something wrong but I think this is the simplest example to get a score out.

When I call `model.score(xTest, yTest)` I get the error `Labels can't be converted to a 1D Tensor`. 

```ts
const btcusdtData = await ensureCryptoData('BTCUSDT', '1d')
const df = new dnf.DataFrame(btcusdtData)
// Couldn't use a list of column names here unfortunately so I used their index numbers. Would be a lot cooler if I could use 'open', 'high', etc.
const x = df.iloc({ columns: [1, 2, 3, 5, 6, 7, 8, 9, 10, 11] })
const y = df.iloc({ columns: [4] })
const [xTrain, xTest, yTrain, yTest] = sk.trainTestSplit(x.tensor, y.tensor)
const model = new sk.LinearRegression()
await model.fit(xTrain, yTrain)
const score = model.score(xTest, yTest)
```

**Comment by dcrescim** [comment]
Hey @Industrial, thanks for reaching out. I think what's happening is your
`const y = df.iloc({ columns: [4] })` line is actually returning a Dataframe to you which has rank 2, instead of rank 1. Meaning it's a 2D matrix with only 1 column.
 
An easy way to check that is to print `y.shape`(you'd likely see an array with 2 values instead of 1). Basically everything that we use as a `y` variable in the whole library needs to be `1D` (either a simple array, or a 1-D Tensor, etc) 

There are functions that will just grab a single column with danfo, namely `df.column("open")` or even just `df['open']` but those require column headers which it looks like you don't have. Another (admittedly hacky) way it to simply take your 2D tensor and convert to a 1D tensor before passing it into `trainTestSplit`, by flattening it with `.reshape`. 
Maybe something like 

`y.tensor.reshape([-1])` instead of just `y.tensor`

Hopefully this helps get you out of the jam, but if not feel free to post again, with your data, and we can go through it together. All the best 😄
